README_LINUX.md

```diff
@@ -1,211 +1,204 @@
+[libsndfile]: http://www.mega-nerd.com/libsndfile
+[fftw]: http://www.fftw.org
+[libavahi-client]: http://www.avahi.org
+[libxt]: http://www.X.org
+
+Recommended packages
+--------------------
+
+For sclang and scide:
+
+- [Qt][Qt] >= 5.7: Cross-platform GUI library, required for the IDE and for sclang's Qt GUI kit. It's best to get the latest Qt 5.x version.
+- QtWebEngine: required for the IDE.
+- [git][git]: Required for sclang's Quarks system.
+- [ALSA][ALSA]: Linux sound library, required for sclang MIDI support.
+- [libudev][libudev]: Device manager library, required for HID support.
+- [Linux kernel][Linux kernel] >= 2.6: Required for LID support.
+- [libreadline][libreadline] >= 5: Required for sclang's CLI interface.
+
+[Qt]: http://qt-project.org
+[ALSA]: http://www.alsa-project.org
+[libudev]: http://www.freedesktop.org/software/systemd/man/libudev.html
+[libreadline]: http://savannah.gnu.org/projects/readline
+[Linux kernel]: http://www.kernel.org
+[git]: https://git-scm.com/
+
+Installing requirements on Debian
+---------------------------------
 
- * gcc >= 4.8
-   http://www.gnu.org/software/gcc
+There are dedicated web pages for building on particular embedded Linux platforms:
 
- * jack and libjack
-   * http://jackit.sourceforge.net
-   * jack audio connection kit
+- [Raspberry Pi](http://supercollider.github.io/development/building-raspberrypi)
+- [BeagleBone Black](https://supercollider.github.io/development/building-beagleboneblack)
 
- * libsndfile >= 1.0
-   * http://www.mega-nerd.com/libsndfile
-   * _the_ soundfile i/o library
+On Debian-like systems, the following command installs the minimal recommended dependencies for compiling scsynth and supernova:
 
- * cmake >= 3.5
-   * http://www.cmake.org
-   * cross-platform build system
+    sudo apt-get install build-essential cmake libjack-dev libjack-jackd2-dev libsndfile1-dev libfftw3-dev libxt-dev libavahi-client-dev
 
- * fftw >= 3.0
-   * http://www.fftw.org
-   * fast FFT transform library (for frequency-domain analysis, phase-vocoder effects)
+The following command installs all the recommended dependencies for sclang except for Qt:
 
- * libxt
-   * http://www.X.org
-   * X toolkit intrinsics
+    sudo apt-get install git libasound2-dev libicu-dev libreadline6-dev libudev-dev pkg-config
 
-Build requirements (optional features)
---------------------------------------
-(most of these will be available in your Linux distribution as packages)
+Installing Qt
+-------------
 
- * Qt >= 5.7 (+ qtwebengine)
-   * http://qt-project.org
-   * cross-platform graphical user interface library, for Qt IDE and sclang's Qt GUI kit
-   * The latest version should work. Tests are done against 5.9.
+**Qt 5.7 or later** is required to be able to run the SuperCollider IDE and sclang's Qt GUI system.
 
- * alsa
-   * http://www.alsa-project.org
-   * advanced Linux sound architecture drivers and library, for sclang's MIDI interface
+### Installing Qt using the official installer
```

Comment:
Since this is the worst case scenario, I'd put this section last, preceded by the PPAs, preceded by "hopefully you're already at Qt5.7+".